client/onboarding-prototype/steps/loading.tsx

```diff
@@ -9,7 +9,7 @@ import apiFetch from '@wordpress/api-fetch';
  * Internal dependencies
  */
 import { useOnboardingContext } from '../context';
-import { EligibleData, EligibleResult } from '../types';
+import { POEligibleData, POEligibleResult } from '../types';
```

Comment:
Nitpick: It seems in the JS code we tend towards camel-case even when using acronyms (e.g. `ApiError` instead of `APIError` - example [here](https://github.com/Automattic/woocommerce-payments/blob/7b437e8716dcf166183a123be4977d20caae0784/client/types/errors.d.ts#L1-L1))

What do you think about using that convention here?

Reply:
Yes, I try to follow the camel-case naming style, but I am split about acronyms like PO or API. In my mind they shouldn't be camel-cased since each letter stands for a whole word, so it shouldn't be uppercase. 

For example: we could have `ProgressiveOnboardingEligibleData` or, the short version, `POEligibleData`. 

This is definitely going into the "mother-of-all philosophy-of-naming debates" 😀, so I can do it both ways, since I don't have a strong stance on it.

Reply:
Yey! I am glad Microsoft agrees :D